packages/dd-trace/test/exporters/agent/writer.spec.js

```diff
@@ -78,7 +92,7 @@ describe('Writer', () => {
       '../../platform': platform,
       '../../log': log,
       '../../format': format,
-      '../../encode': encode,
+      '../../encode/index': encode,
```

Comment:
🧐

Reply:
It needs to be stubbed directly like this. Proxyrequire didn't do the job otherwise. 🤷‍♂